src/lib.rs

```diff
@@ -90,9 +93,6 @@ pub mod trufflesuite_ganachecli;
 #[cfg(feature = "victoria_metrics")]
 #[cfg_attr(docsrs, doc(cfg(feature = "victoria_metrics")))]
 pub mod victoria_metrics;
-#[cfg(feature = "zookeeper")]
-#[cfg_attr(docsrs, doc(cfg(feature = "zookeeper")))]
-pub mod zookeeper;
```

Comment:
Why zookeeper is removed? I guess not intentionally committed?

Reply:
Readded.